lte/gateway/c/core/oai/tasks/sgw/pgw_handlers.cpp

```diff
@@ -437,13 +443,13 @@ status_code_e spgw_handle_nw_initiated_bearer_deactv_req(
 }
 
 // Send ITTI message,S11_NW_INITIATED_DEACTIVATE_BEARER_REQUEST to mme_app
-int32_t spgw_build_and_send_s11_deactivate_bearer_req(
+status_code_e spgw_build_and_send_s11_deactivate_bearer_req(
     imsi64_t imsi64, uint8_t no_of_bearers_to_be_deact,
     ebi_t* ebi_to_be_deactivated, bool delete_default_bearer,
     teid_t mme_teid_S11, log_proto_t module) {
   OAILOG_FUNC_IN(module);
   MessageDef* message_p = itti_alloc_new_message(
-      module, S11_NW_INITIATED_DEACTIVATE_BEARER_REQUEST);
```

Comment:
This function is used by both SPGW task and SGW_S8 task, so need to specify the right sender task based on the `module` parameter.